.github/workflows/cicd.yml

```diff
@@ -48,5 +48,5 @@ jobs:
         with:
           channel: 'stable'
       - uses: bluefireteam/melos-action@main
-      - run: melos run test
+      - uses: bluefireteam/spec-action@main
```

Comment:
Oh, is this new?

Reply:
Yep, created it as an experiment like half an hour ago :smile:

Reply:
(it's pretty much exactly the same as the melos-action)

Reply:
pretty cool, we should make a tweet

Reply:
Good idea!
It's here: https://github.com/marketplace/actions/spec-action